ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -270,7 +299,19 @@ pattern GenTxMary tx = HardForkGenTx (OneEraGenTx (TagMary tx))
 pattern GenTxAlonzo :: GenTx (ShelleyBlock (TPraos c) (AlonzoEra c)) -> CardanoGenTx c
 pattern GenTxAlonzo tx = HardForkGenTx (OneEraGenTx (TagAlonzo tx))
 
-{-# COMPLETE GenTxByron, GenTxShelley, GenTxAllegra, GenTxMary, GenTxAlonzo #-}
+pattern GenTxBabbage :: GenTx (ShelleyBlock (Praos c) (BabbageEra c)) -> CardanoGenTx c
+pattern GenTxBabbage tx = HardForkGenTx (OneEraGenTx (TagBabbage tx))
+
+{-# COMPLETE
```

Comment:
Style: let's use leading `,`s, so that just have to add one line each time (and match the existing COMPLETE pragmas in this module)